### PR TITLE
Basic changes for Jenkins

### PR DESCRIPTION
**Pull request by techgnosis**
* Jenkinsfile
* kubectl namespacing
* whitespace

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -64,37 +64,118 @@ spec:
     }
   }
 
+  environment {
+    GOOGLE_APPLICATION_CREDENTIALS    = '/home/jenkins/dev/jenkins-deploy-dev-infra.json'
+  }
 
   stages {
```
Comment (chrislovecnm) [inline Jenkinsfile:73]: Total nit pick, but can we please get some code level documentation in for the different stages.  Thanks

**Review by chrislovecnm — CHANGES_REQUESTED** [review]
Looks great overall. Just would love some code level docs in the Jenkinsfiles.

**Comment by techgnosis** [comment]
test this please

**Review by chrislovecnm — APPROVED** [review]

**Pull request merged**